frontend/src/components/Register/Register.js

```diff
@@ -1,59 +1,52 @@
-import React from "react";
-import axios from "axios";
-import { useState } from "react";
 import '../../config';
+import axios from "axios";
+import React, { useState } from "react";
+
 import './Register.scss'
 
-export const Register = (props) => {
+export default function Register(props) {
 
     const [errorMsg, setErrorMessage] = useState("");
     
-    // Submit registration form
     const submit = (event) => {
```

Comment:
Can we give this a more representative name? Seeing lots of functions named "submit" lately and although the file they are in gives them context, I think naming it a bit more clearly can go a long way

Reply:
Yes, definitely! I had also noticed that trend... I renamed it `submitRegistrationForm` :)